variables.tf

```diff
@@ -101,6 +147,21 @@ DESCRIPTION
   nullable    = false
 }
 
+variable "inbound_ip_rules" {
+  type = list(object({
+    ip_mask = string                    # Required - The IP mask (CIDR) to match on
+    action  = optional(string, "Allow") # Optional - The action to take when the rule is matched
+  }))
+  default     = null
+  description = "(Optional) One or more inbound_ip_rule blocks as defined below."
+}
+
+variable "is_zone_redundant" {
```

Comment:
This variable currently does nothing due to the merge. If it is not required, please remove it.